tests/unit/best_practices.cpp

```diff
@@ -2378,4 +2378,23 @@ TEST_F(VkBestPracticesLayerTest, NonSimultaneousSecondaryMarksPrimary) {
     vk::CmdExecuteCommands(m_commandBuffer->handle(), 1, &secondary.handle());
     m_errorMonitor->VerifyFound();
     m_commandBuffer->end();
-}
\ No newline at end of file
+}
+
+TEST_F(VkBestPracticesLayerTest, NoCreateSwapchainPresentModes) {
+    TEST_DESCRIPTION("With swapchain maintenance 1, CreateSwapchain with VkPresentModesCreateInfoEXT");
+
+    AddSurfaceExtension();
+    AddRequiredExtensions(VK_EXT_SWAPCHAIN_MAINTENANCE_1_EXTENSION_NAME);
+    ASSERT_NO_FATAL_FAILURE(InitBestPracticesFramework());
+    if (!AreRequiredExtensionsEnabled()) {
+        GTEST_SKIP() << RequiredExtensionsNotSupported() << " not supported.";
+    }
+
+    ASSERT_NO_FATAL_FAILURE(InitState());
+    if (!InitSurface()) {
+        GTEST_SKIP() << "Cannot create surface";
+    }
+    m_errorMonitor->SetDesiredFailureMsg(kWarningBit, kVUID_BestPractices_NoVkSwapchainPresentModesCreateInfoEXTProvided);
```

Comment:
Sorry took me so long to respond.

`scripts/vk_validation_stats.py` is crashing here (Unhelpfully). The fix is to use the underlying string.

```
 python scripts/vk_validation_stats.py external/Debug/64/Vulkan-Headers/build/install/share/vulkan/registry/validusage.json -export_header
Unable to categorize VUID: 
Confused while parsing VUIDs in test code - cannot proceed. (FIXME)
```

```suggestion
    m_errorMonitor->SetDesiredFailureMsg(
        kWarningBit, "UNASSIGNED-BestPractices-vkCreateSwapchainKHR-no-VkSwapchainPresentModesCreateInfoEXT-provided");
```